ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -819,4 +819,153 @@
               case-sensitive: true
               default-excludes: true
               fingerprint: false
+              only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
+          test_name: rancher-e2e
+          node: 'antrea-rancher-testbed'
```

Comment:
yaml lines are repeated across this file, you can use anchor block to define common attribute at one place and use it across other places with anchor and alias

Reply:
this is the job template so for every job it has to be there